exchanges/stream/stream_match.go

```diff
@@ -44,8 +44,8 @@ func (m *Match) IncomingWithData(signature interface{}, data []byte) bool {
 	return false
 }
 
-// Sets the signature response channel for incoming data
-func (m *Match) set(signature interface{}) (matcher, error) {
+// Set the signature response channel for incoming data
+func (m *Match) Set(signature interface{}) (matcher, error) { //nolint:revive // Yes: matcher is private, but .C is public and usable
```

Comment:
nolint because:
`unexported-return: exported method Set returns unexported type stream.matcher, which can be annoying to use (revive)`

Reply:
Fixed below by making matcher public